environments/india/app-processes.yml

```diff
@@ -64,6 +64,7 @@ pillows:
     case-pillow:
       num_processes: 2
       dedicated_migration_process: True
+      processor_chunk_size: 0
```

Comment:
right, but why is that preferable to a batch size of 1? This isn't blocking, but it feels like they are equivalent, and we should be consistent in which we use. Right now, we mostly use 1, but if this is better we should change that for all of them. If it is not better, we should stick to 1.

Reply:
I could change all to 0 though as a follow up in a different PR.

Reply:
> having one record defeats the purpose of a "batch". So it's better to say that we don't want batch processing at all rather than misusing batch of 1.

Sounds reasonable to me, although if that is the case it would be great to change the other two in this file (and the ones on staging and swiss). That can be a followup if you prefer though.

Reply:
Thanks @calellowitz 
I will update just the case-pillow on india for now and then take rest as follow ups for staging and india